doc/catalog-format-spec.md

```diff
@@ -58,8 +58,8 @@ libbar = "^2.0 & /=2.1.3" # Excluding a known bad version
 In some contexts, information can be dynamic: special encodings can be used to
 make data vary depending on the environment (OS, architecture, ...). The
 environment is represented as a set of specific variables which can have a
-specific set of values: see the Parameters section below for a comprehensive
-list.
+specific set of values: see the [Parameters](#-parameters) section below for a
+comprehensive list.
```

Comment:
This link does not work for me, I think because of the '-' after the '#' which doesn't seem to be removed in the rendered file.

Reply:
I see it online too. I tried to follow advice found online. Will try without the space as I cannot test locally (unless you have a way!)